.github/workflows/vol_rest.yml

```
@@ -4,7 +4,7 @@ on:
   workflow_call:
     inputs:
       build_mode:
-        description: "Build type"
+        description: "CMake Build type"
```

CMake descriptor was removed from CI  names, descriptions, etc. in PR #5813, since CMake is now our only build type.
```suggestion
        description: "Build type"
```